crates/verify/src/utils.rs

```diff
@@ -189,7 +189,7 @@ fn is_partial_match(
     // 1. Check length of constructor args
     if constructor_args.is_empty() || is_runtime {
         // Assume metadata is at the end of the bytecode
-        return try_extract_and_compare_bytecode(local_bytecode, bytecode)
+        return try_extract_and_compare_bytecode(local_bytecode, bytecode);
```

Comment:
Please undo the unrelated formatting changes

Reply:
Use nightly to format: `cargo +nightly fmt`

Reply:
Reformatted using `cargo +nightly fmt`, thanks guys!